fuzz/fuzz_targets/utils.rs

```diff
@@ -1,71 +1,48 @@
 #![allow(dead_code)]
 
 use arbitrary::{Arbitrary, Unstructured};
-use wasmi::{core::ValType, Val};
-
-pub fn exec_config() -> wasm_smith::Config {
-    wasm_smith::Config {
-        export_everything: true,
-        allow_start_export: false,
-        reference_types_enabled: false,
-        max_imports: 0,
-        max_memory32_bytes: (1 << 16) * 1_000,
-        // Note: wasmi does not support 64-bit memory, yet.
-        memory64_enabled: false,
-        max_data_segments: 10_000,
-        max_element_segments: 10_000,
-        max_exports: 10_000,
-        max_elements: 10_000,
-        min_funcs: 1,
-        max_funcs: 10_000,
-        max_globals: 10_000,
-        max_table_elements: 10_000,
-        max_values: 10_000,
-        max_instructions: 100_000,
-        exceptions_enabled: false,
-        simd_enabled: false,
-        threads_enabled: false,
-        gc_enabled: false,
-        tail_call_enabled: false,
-        ..Default::default()
-    }
+use wasmi::{core::ValType, ExternRef, FuncRef, Val};
+
+pub fn disable_unsupported_config(config: &mut wasm_smith::Config) {
+    config.gc_enabled = false;
+    config.exceptions_enabled = false;
+    config.max_memories = 1;
```

Comment:
Wasmi supports `multi-memory` since v0.37.0. :) Can be enabled.

Reply:
The WasmiStack DifferentialTarget also has issues creating a module when this is enabled.

Reply:
true.
Wasmtime supports it though.
Ideally our fuzzing strategy was a bit more flexible and depended on `Unstructured` input what other Wasm runtime to compare with `wasmi_reg`.